paasta_tools/paasta_remote_run.py

```diff
@@ -136,5 +181,80 @@ def handle_interrupt(_signum, _frame):
     t.join(float("inf"))
 
 
+def remote_run_stop(args):
+    paasta_config, service, cluster, soa_dir, instance, instance_type = extract_args(args)
+    # native_config = paasta_config.get_paasta_native_config()
+    # credential = mesos_pb2.Credential()
+    # credential.principal = native_config()['principal']
+    # credential.secret = native_config()['secret']
+    # mesos_uri = '%s:%d' % (mesos_tools.get_mesos_leader(), mesos_tools.MESOS_MASTER_PORT)
+    framework_id = args.framework_id
+    if framework_id is None:
+        if args.run_id is not None:
+            if re.match('\s', args.run_id):
+                paasta_print(PaastaColors.red("Run id must not contain whitespace."))
+                os._exit(1)
+
+            frameworks = requests.get('http://localhost:5050/frameworks').json()['frameworks']
+            found = [f for f in frameworks if re.search(' %s$' % args.run_id, f['name']) is not None]
+            if len(found) > 0:
+                framework_id = found[0]['id']
+            else:
+                paasta_print(PaastaColors.red("Framework with run id %s not found." % args.run_id))
+                os._exit(1)
+        else:
+            paasta_print(PaastaColors.red("Must provide either run id or frameowk id to stop."))
+            os._exit(1)
+    else:
+        frameworks = requests.get('http://localhost:5050/frameworks').json()['frameworks']
+        found = [f for f in frameworks if f['id'] == framework_id]
+        if len(found) > 0 and re.search('^paasta-remote %s.%s ' % (service, instance), found[0]['name']) is None:
+            paasta_print(
+                PaastaColors.red(
+                    "Framework name %s does not match remote-run service instance %s.%s" %
+                    (found[0]['name'], service, instance)))
+            os._exit(1)
+
+    paasta_print("Tearing down framework %s." % framework_id)
+    teardown = requests.post('http://localhost:5050/teardown', data="frameworkId=%s" % framework_id)
+    if teardown.status_code == 200:
+        paasta_print(PaastaColors.green("OK"))
+    else:
+        paasta_print(teardown.text)
+
+
+def remote_run_list(args):
+    paasta_config, service, cluster, soa_dir, instance, instance_type = extract_args(args)
+    # native_config = paasta_config.get_paasta_native_config()
+    # credential = mesos_pb2.Credential()
+    # credential.principal = native_config()['principal']
+    # credential.secret = native_config()['secret']
+    # mesos_uri = '%s:%d' % (mesos_tools.get_mesos_leader(), mesos_tools.MESOS_MASTER_PORT)
+    frameworks = requests.get('http://localhost:5050/frameworks').json()['frameworks']
```

Comment:
could you add these functions to the mesos api client instead if they're missing, or use that instead if not? code is [here](https://github.com/Yelp/paasta/blob/master/paasta_tools/mesos/master.py)